src/main/java/org/snomed/snowstorm/core/data/services/ConceptUpdateHelper.java

```diff
@@ -606,7 +606,7 @@ private void doSaveBatchQueryConcept(Collection<QueryConcept> queryConcepts, Com
 			
 			
 			//Any change to a component in an extension needs to be done in the default module
-			if (newComponent.isChanged() && (defaultModuleId != null && 
+			if (!rebase && newComponent.isChanged() && (defaultModuleId != null &&
 					!defaultModuleId.equals(Concepts.CORE_MODULE))) {
 				newComponent.setModuleId(defaultModuleId);
 			}
```

Comment:
`newComponent` will be flagged as `changed` whenever doing a rebase. As a result, `newComponent` is having its module changed on a rebase. This caused Descriptions to lose their effectiveTime / move module for one of my tests.

Reply:
Can you please add a test case for this change?

Reply:
If I remove `!rebase` or `newComponent.isComponentChanged(existingComponent)` then two tests fail: 

`testUpgradeExtensionWhenExtensionProjectAndTaskBothHaveAuthoredSameComponentAndUserSelectsMiddle` and `testUpgradeExtensionWhenExtensionProjectAndTaskBothHaveAuthoredSameComponentAndUserSelectsRHS`. 

These fails because the `effectiveTime` and `module` have incorrectly changed for some Components.